src/components/Books.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/jsx-key */
+import React from 'react';
+import Book from './Book';
+
+export default function User() {
```

Comment:
- [ ] OPTIONAL - Consider changing this function name to describe the actual functionality of the function.